src/_static/tutorial/cylc-forecasting-suite/bin/post-process

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (C) 2012-2019 British Crown (Met Office) & Contributors - GNU V3+.
+# This is illustrative code developed for tutorial purposes, it is not
+# intended for scientific use and is not guarantied to be accurate or correct.
+# -----------------------------------------------------------------------------
+"""
+Usage:
+    post-process SITE TIME
+
+Arguments:
+    SITE: The name of the location to process data for.
```

Comment:
Should these args be moved to the `main` doc? Only asking because my IDE does not understand these are arguments passed to this script - if it's a convention in Python scripts called directly from command line, then feel free to ignore/resolve :slightly_smiling_face:

Reply:
Not sure about convention, this is usage documentation relevant to the script itself. Could move if wanted.